docs/edge/monitoring.rst

```diff
@@ -0,0 +1,250 @@
+.. _edge-monitoring:
+
+Monitor CrateDB Edge
+====================
+
+This tutorial introduces ways to set up monitoring of your CrateDB Edge cluster.
+Visualization tool `Grafana`_ is used together with `Prometheus`_ and `Loki`_
+to monitor the cluster performance and browse logs, respectively.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+.. _edge-monitoring-prereqs:
+
+Prerequisites
+-------------
+
+- The CrateDB Edge installation script requires Helm to be present in the system.
+- For a deployment with Grafana, Loki, and Prometheus, we recommend at
+  least 5 CPUs and 5 GiB of memory per node for acceptable performance.
+
+.. _edge-monitoring-deployment:
+
+Cluster Deployment
+------------------
+
+Deployment steps will depend on your :ref:`cloud provider <edge-providers>`,
+or on your local environment, if you choose the :ref:`self-hosted
+<edge-self-hosted>` variant of CrateDB Edge.
+
+.. _edge-monitoring-grafana-install:
+
+Grafana Install
+---------------
+
+The first important step comes after the successful installation of CrateDB into your
+chosen system. Right after you get the following console prompt:
+
+.. code-block:: console
+
+  ✓ Region is connected to CrateDB Cloud.
+  ✓ Successfully validated installation
+
+This means that base installation is done. You will now be asked if you want to
```

Comment:
Should maybe be clarified, that the install script can also be run again to get to those steps, in case one missed them the first time.